cmd/oceantv/broadcast_hardware_machine.go

```diff
@@ -247,6 +259,20 @@ func (sm *hardwareStateMachine) handleHardwareResetRequestEvent(event hardwareRe
 	}
 }
 
+func (sm *hardwareStateMachine) handleControllerFailureEvent(event controllerFailureEvent) {
+	sm.log("handling controller failure event")
+	switch sm.currentState.(type) {
+	case *hardwareOn, *hardwareRestarting, *hardwareStarting:
+		sm.transition(newHardwareStopping(sm.ctx))
+	case *hardwareOff:
+		sm.transition(newHardwareStarting(sm.ctx))
+	case *hardwareStopping:
```

Comment:
I'm thinking this should go up into the same case as on, restarting and starting.

Reply:
Done.